### ISSUE TITLE
Can ActionListUUP be used with Windows 11's UUPs?

### ISSUE THREAD

**Issue by DellZHackintosh** [abbodi1406/BatUtil#44]
ActionListUUP has not updated for 5 years. I wonder that is it workable on Windows 11, in order to update without the Windows 11 Hardware Compatibility Detection.

**Comment by abbodi1406** [comment]
I never tried it for Windows 11, and it probably won't work
even if it worked, i doubt it can bypass any HW requirements

**Comment by DellZHackintosh** [comment]
The only thing that we need to worry about is Windows Update - without this we can never update with uups.
Once we can update with uups manually, the rest of requirements can be bypassed easily.
BTW, the script seems that ignores any `Windows11.0*` file. Not sure if it's the only obstacle. 🤔

**Comment by DellZHackintosh** [comment]
Looks like there're actually some changes for `ActionList.xml`, such as the reference for `*.psf`.
Here's my ActionList(from 23H2 to 24H2): 
[ActionList.zip](https://github.com/user-attachments/files/17238982/ActionList.zip)

**Comment by abbodi1406** [comment]
Thanks

that would be difficult to construct manually
maybe i can use UpdateAgent.dll `UA_CreateActionList` function, if i can figure the parameters

**Comment by DellZHackintosh** [comment]
> i doubt it can bypass any HW requirements

Yes, it can be - `/product server` works fine with WindowsUpdateBox too, and it's well known that upgrading by this way have absolute advantages comparing with ISO
